Library/Homebrew/rubocops/bottle_block_cop.rb

```diff
@@ -0,0 +1,36 @@
+module RuboCop
+  module Cop
+    module CustomCops
```

Comment:
```module Homebrew``` has been used in ```Library/Homebrew/cmd/``` and ```Library/Hombrew/dev-cmd/``` , so I thought calling it as ```Homebrew``` might cause some sort of confusion/clash ( I am not sure) and named it as ```CustomCops```